test/fixtures/v1-vt-05401-state-utility-lowincome.json

```diff
@@ -548,85 +701,109 @@
       "owner_status": [
         "homeowner"
       ],
-      "start_date": "2020",
+      "start_date": "2020-01-01",
       "ami_qualification": null,
-      "eligible": true,
+      "eligible": false,
```

Comment:
This is EVT's income bonus to air-to-water HPs. It's ineligible because EVT's base air-to-water incentive is excluded by BED's.